### PR TITLE
Fix proj-taskcluster/aws-test live logs

### PR DESCRIPTION
**Pull request by petemoore**
PR #422 fixed docker-worker livelogs for our Google Cloud docker-worker workers - which covered all worker pools except for [`proj-taskcluster/aws-test`](https://community-tc.services.mozilla.com/worker-manager/proj-taskcluster%2Faws-test).

This PR updates our AWS images too, which are used by [`proj-taskcluster/aws-test`](https://community-tc.services.mozilla.com/worker-manager/proj-taskcluster%2Faws-test). Moreover, this is useful for verifying that `imageset.sh` can be used to patch existing images worker images in both AWS and Google Cloud, which will be needed for fixing firefox-ci workers too in [bug 1714818](https://bugzil.la/1714818).

Changes have been applied (already) with:

```
# build new AMIs in AWS/EC2
imagesets/imageset-under-docker.sh aws update docker-worker  

# update worker pools to use them
TASKCLUSTER_ROOT_URL=https://community-tc.services.mozilla.com tc-admin apply
```

### CODE REVIEW

**Comment by petemoore** [comment]
Running a test task [here](https://community-tc.services.mozilla.com/tasks/VFftNe4eSXKHy3uYY0Y3_A) to confirm that live logs work now.

**Comment by petemoore** [comment]
The live logging in task [X3Mu6_98RrWwnj9tgdRM7Q](https://community-tc.services.mozilla.com/tasks/X3Mu6_98RrWwnj9tgdRM7Q/runs/0/logs/live/public/logs/live.log) worked!

File: generate/workers.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def aws(
       regions: regions to deploy to (required)
       instanceTypes: dict of instance types to provision, values are
                      capacityPerInstance (required)
-      securityGroup: name of the security group to appply (default no-inbound)
+      securityGroup: name of the security group to apply (default no-inbound)
```
Comment (jwhitlock) [inline generate/workers.py:335]: ❤️

File: imagesets/docker-worker/aws_base_instance_type [hunk 2]
```diff
@@ -0,0 +1 @@
+c4.2xlarge
```
Comment (jwhitlock) [inline imagesets/docker-worker/aws_base_instance_type:1]: 👍, the m3.2xlarge size is deprecated and giving us frequent errors when we're out of spot instances

**Review by jwhitlock — APPROVED** [review]
Thanks @petemoore!

**Pull request merged**